test/tools/libtesteth/ImportTest.cpp

```diff
@@ -56,6 +56,114 @@ ImportTest::ImportTest(json_spirit::mObject const& _input, json_spirit::mObject&
 	importState(_input.at("pre").get_obj(), m_statePre);
 }
 
+void ImportTest::makeBlockchainTestFromStateTest(vector<eth::Network> const& _networks) const
+{
+    // Generate blockchain test filler
+    string testnameOrig = TestOutputHelper::get().testName();
+    for (auto& tr : m_transactions)
+    {
+        json_spirit::mObject json;
+        json_spirit::mObject testObj;
+
+        // generate the test name with transaction detail
+        string postfix = "_d" + toString(tr.dataInd);
+        postfix += "g" + toString(tr.gasInd);
+        postfix += "v" + toString(tr.valInd);
+        string testname = testnameOrig + postfix;
+
+        // basic genesis
+        json_spirit::mObject genesisObj = TestBlockChain::defaultGenesisBlockJson();
+        genesisObj["coinbase"] = toString(m_envInfo->author());
+        genesisObj["gasLimit"] = toCompactHexPrefixed(m_envInfo->gasLimit());
+        genesisObj["timestamp"] = toCompactHexPrefixed(m_envInfo->timestamp() - 50);
+        testObj["genesisBlockHeader"] = genesisObj;
+        testObj["pre"] = fillJsonWithState(m_statePre);
+
+        // generate expect sections for this transaction
+        if (m_testInputObject.count("expect"))
+        {
+            State s = State(0, OverlayDB(), eth::BaseState::Empty);
+            AccountMaskMap m = std::unordered_map<Address, AccountMask>();
```

Comment:
Are you sure you need ` = ...;`?

I guess `AccountMaskMap m;` would initialize `m` to be empty.